ua-snap/arctic-eds
# Safe mode

To test:

`export EDS_SAFE_MODE=True`

Items to check:

- Home page should have a warning in the intro text
- Home page should not show the "pick a place" controls, instead it shows another repeated note about maintenance and a list of places to pick from
- Click on each place: all should load reports + look OK
- On the reports page, just above the Table of Contents there's another warning that "CSV and links to source data not available."
- Modules should not be linked to anywhere
- CSV downloads should not be linked to anywhere
- Links to our data catalog should not be visible anywhere
- The Map link in the navbar should be removed
- If you are coy and go to `/maps` you should get a blank page
